android/src/main/java/com/cloudwebrtc/webrtc/MethodCallHandlerImpl.java

```diff
@@ -1201,16 +1201,17 @@ public void mediaStreamAddTrack(final String streamId, final String trackId, Res
       if (track != null) {
         if (track.kind().equals("audio")) {
           mediaStream.addTrack((AudioTrack) track);
+          result.success(null);
         } else if (track.kind().equals("video")) {
           mediaStream.addTrack((VideoTrack) track);
+          result.success(null);
         }
       } else {
```

Comment:
To provide result in all possible flow cases I propose to add:
```java
String kind = track.kind();
...
resultError("mediaStreamRemoveTrack", "mediaStreamAddTrack() track [" + trackId + "] has unsupported type: " + kind, result);
```
between this lines.

Reply:
Or maybe move `result.success(null);` to that place. But as for me error will more relevant.

Reply:
Makes sense, I added the error message.